webcompat/static/js/lib/bugform.js

```diff
@@ -573,6 +605,136 @@ function BugForm() {
     });
   };
 
+  // functions for Image Editor
+  this.openImageEditor = function() {
+    //var doc = global.document;
+    var canvas = document.createElement('canvas');
```

Comment:
At some point we need to destroy this `canvas` element, otherwise we'll end up just making more elements if you open and close the editor multiple times.